app/views/facility_reservations/_instrument.html.haml

```diff
@@ -1,25 +1,19 @@
 .timeline_instrument[instrument]
   %h4
-    - if public_timeline?
-      = link_to instrument,
-        facility_instrument_path(current_facility, instrument)
-    - else
-      = link_to instrument,
-        facility_instrument_schedule_path(current_facility, instrument)
-
+    = link_to instrument, facility_instrument_schedule_path(current_facility, instrument)
```

Comment:
Are public/admin-only timelines not a thing anymore?

Reply:
They are! The facility timelines are `FacilityReservations#timeline`, and the public timelines are `Reservations#public_timeline`. This partial is `facility_reservations/_instrument.html.haml`, so the conditionals are gone because we know exactly which context we’re rendering from.

I find this to be much better than putting partials in `shared` and then using ivars like `@public_timeline = true` to set the context — that suggests that the partial is not really shared, since its content depends on the context where it is rendered, so it’s simpler to just split the content into unconditioned partials 😄 Does that make sense?